docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/BinaryClassification.ttinclude

```diff
@@ -56,7 +64,13 @@ namespace Samples.Dynamic.Trainers.BinaryClassification
 
             // Evaluate the overall metrics
             var metrics = mlContext.BinaryClassification.<#=Evaluator#>(transformedTestData);
-            Microsoft.ML.SamplesUtils.ConsoleUtils.PrintMetrics(metrics);
+            Console.WriteLine($"Accuracy: {metrics.Accuracy:F2}");
+            Console.WriteLine($"AUC: {metrics.AreaUnderRocCurve:F2}");
+            Console.WriteLine($"F1 Score: {metrics.F1Score:F2}");
+            Console.WriteLine($"Negative Precision: {metrics.NegativePrecision:F2}");
+            Console.WriteLine($"Negative Recall: {metrics.NegativeRecall:F2}");
+            Console.WriteLine($"Positive Precision: {metrics.PositivePrecision:F2}");
+            Console.WriteLine($"Positive Recall: {metrics.PositiveRecall:F2}");
```

Comment:
thanks for changing this. I think, it would be cleaner to replicate ConsoleUtils.PrintMetrics as a helper function PrintMetrics at the bottom of the .ttinclude file, so that Example() becomes leaner.